pkg/rsyslog/Dockerfile.in

```diff
@@ -0,0 +1,15 @@
+FROM alpine:3.8
+WORKDIR /
+COPY rsyslog@lists.adiscon.com-5a55e598.rsa.pub /etc/apk/keys/rsyslog@lists.adiscon.com-5a55e598.rsa.pub
+# hadolint ignore=DL3018
+RUN echo "http://alpine.rsyslog.com/3.7/stable" >> /etc/apk/repositories \
+            && apk add --no-cache \
+            rsyslog \
+            rsyslog-imptcp \
+            rsyslog-mmjsonparse \
+            rsyslog-omstdout
```

Comment:
If you move to upstream 3.10.3 Alpine -- you will no longer need separate rsyslog-imptcp & rsyslog-omstdout

Reply:
Done